### ISSUE TITLE
Document "call-with-web-prompt"?

### ISSUE THREAD

**Issue by LiberalArtist** [racket/web-server#18]
Should `call-with-web-prompt` be a documented export of `web-server/lang/abort-resume`?

It is mentioned [here](https://groups.google.com/d/msg/racket-users/KWLcu8XgOHM/QhI5v3RbKRUJ), and I have used it at least once — but I'm asking rather than documenting as I'm not sure if it's intended to be a "documented" part of the API, with everything that entails.

**Comment by jessealama** [comment]
@LiberalArtist are you satisfied with Jay’s answer?

**Comment by LiberalArtist** [comment]
I think so.

What I had in mind when I opened this issue (I think: I haven't thought about it much in the last two years) was re-using the serializable continuation infrastructure in contexts that have nothing to do with the web, as was discussed in the mailing-list thread I linked to. I still think that would be a nice thing in a world with infinite time, but it seems like a lot of work to do it properly. In a way, it's a happy accident that `call-with-web-prompt` is exported to allow for some experimentation ([`live-free-or-die!`](https://docs.racket-lang.org/live-free-or-die/)) but it's clearly a hack that could subvert the invariants of `#lang web-server`, so I think I agree that documenting it seems like a bad idea.

So I guess I'll close this issue.

**Comment by jessealama** [comment]
My understanding was that @joergen7 wanted to do something similar (namely, use serializable continuations outside of the web server). (Perhaps I'm getting the details wrong...)

**Comment by joergen7** [comment]
TL;DR: my interest is not so much about serializable *continuations* as it is about serializable *lambdas*.

I tried to use `serial-lambda` to find a way around the limitations of place channels. Place channels cannot transmit normal lambdas. But in principle they can transmit serial-lambdas.

Only that things blow up in my face when I try. I will create a proper issue to discuss the topic.
